### PR TITLE
feature1added

### PR DESCRIPTION
**Pull request by venky-kumar**
[*] Displaying the rental vehicles

### CODE REVIEW

File: src/main/java/in/venkatet/service/VehicleService.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	 * method to display the products available
+	 * 
+	 * @return
+	 * 
+	 */
+	public static Map<String, Integer> getProducts() {
+		return vehicles;
+	}
+
+	public static boolean addProduct(String vehicleName, int cost) {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/venkatet/service/VehicleService.java]: Choose appropriate method

File: src/main/java/in/venkatet/servlet/AddVehicleServlet.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+		int cost = Integer.parseInt(request.getParameter("cost"));
+		boolean validCost = VehicleService.isValidNumber(cost);
+
+		boolean isAdded = false;
+		if (validCost) {
+			try {
+				isAdded = VehicleService.addProduct(vehicleName, cost);
+			} catch (Exception e) {
+				String errorMessage = "Invalid data ";
+				response.sendRedirect("addproduct.jsp?errorMessage=" + errorMessage);
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/venkatet/servlet/AddVehicleServlet.java]: Add Product - irrelevant page name

File: src/main/java/in/venkatet/servlet/AddVehicleServlet.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+			}
+
+		}
+		if (isAdded) {
+			try {
+				response.sendRedirect("Display.jsp");
+			} catch (Exception e) {
+				String errorMessage = "Unable to add Books ";
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/venkatet/servlet/AddVehicleServlet.java]: Irrelevant errorMessage

**Review by ghost — CHANGES_REQUESTED** [review]
Review comments added

**Pull request merged**